examples/using-preact/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Hello World example
+
+## How to use
+
+Download the example (or clone the repo)[https://github.com/zeit/next.js.git]:
```

Comment:
I just copied this from the hello-world example. May be we need to change this for all examples. Shall we do it with another PR?